lms/static/scripts/frontend_apps/components/test/ErrorDisplay-test.js

```diff
@@ -54,60 +64,40 @@ describe('ErrorDisplay', () => {
       .filterWhere(n => n.text() === 'open a support ticket');
 
     const url = new URL(link.prop('href'));
-    assert.equal(
-      url.searchParams.get('subject'),
-      '(LMS Error) Failed to fetch files: A message from the back end'
-    );
+
     assert.include(
       url.searchParams.get('content'),
       'Error code: some_backend_error_code'
     );
     assert.include(
       url.searchParams.get('content'),
-      '"foo": "bar"',
+      'foo: bar',
       'Includes details'
     );
   });
 
-  [
-    { description: '' },
-    { description: 'Oh no', details: null },
-    { description: 'Oh no', details: '' },
-  ].forEach(error => {
-    it('omits technical details if not provided', () => {
-      const wrapper = mount(
-        <ErrorDisplay
-          message="Something went wrong"
-          error={{ message: '', details: error.details }}
-        />
-      );
+  it('omits technical details if not provided', () => {
+    fakeFormatErrorDetails.returns('');
+    const wrapper = mount(
+      <ErrorDisplay message="Something went wrong" error={{ message: '' }} />
+    );
 
-      const details = wrapper.find('pre');
-      assert.isFalse(details.exists());
-    });
+    const details = wrapper.find('[data-testid="error-details"]');
+    assert.isFalse(details.exists());
   });
 
-  [
-    {
-      details: 'Note from server',
-      expectedText: 'Note from server',
-    },
-    {
-      details: { statusCode: 123 },
-      expectedText: '{ "statusCode": 123 }',
-    },
-  ].forEach(({ details, expectedText }) => {
-    it('displays technical details if provided', () => {
-      const error = { message: '', details };
-
-      const wrapper = mount(
-        <ErrorDisplay message="Something went wrong" error={error} />
-      );
-
-      const detailsEl = wrapper.find('pre');
-      assert.isTrue(detailsEl.exists());
-      assert.include(detailsEl.text().replace(/\s+/g, ' '), expectedText);
-    });
+  it('displays technical details if provided', () => {
+    const details = 'Some details';
+    fakeFormatErrorDetails.returns(details);
+    const error = { message: '', details };
+
+    const wrapper = mount(
+      <ErrorDisplay message="Something went wrong" error={error} />
+    );
+
+    const detailsEl = wrapper.find('[data-testid="error-details"]');
+    assert.isTrue(detailsEl.exists());
```

Comment:
In this test and others, it might be worth adding assertions that the correct error is passed to `formatErrorDetails` and `formatErrorMessage`.